spec/acceptance/suites/fips_from_fixtures/00_default_spec.rb

```diff
@@ -0,0 +1,63 @@
+class ScrubFixtures
+  require 'simp/beaker_helpers'
+  include Simp::BeakerHelpers
+
+  def initialize
+    FileUtils.rm_rf(File.join(fixtures_path, 'modules'))
+  end
+end
+
+require 'yaml'
+require 'tempfile'
+
+@alt_fixtures = '.fips_fixtures.yml'
+
+new_fixtures = {
+  'fixtures' => {
+    'repositories' => {}
+  }
+}
+
+new_fixtures['fixtures']['repositories']['fips'] = 'https://github.com/simp/pupmod-simp-fips'
+new_fixtures['fixtures']['repositories']['augeasproviders_core'] = 'https://github.com/simp/augeasproviders_core'
+new_fixtures['fixtures']['repositories']['augeasproviders_grub'] = 'https://github.com/simp/augeasproviders_grub'
+new_fixtures['fixtures']['repositories']['simplib'] = 'https://github.com/simp/pupmod-simp-simplib'
+new_fixtures['fixtures']['repositories']['stdlib'] = 'https://github.com/simp/puppetlabs-stdlib'
+
+File.open(@alt_fixtures, 'w'){ |fh| fh.puts(new_fixtures.to_yaml) }
+
+ScrubFixtures.new
+
+ENV['BEAKER_fips'] = 'yes'
+ENV['FIXTURES_YML'] = @alt_fixtures
+
+Bundler.with_clean_env{
+  ENV['FIXTURES_YML'] = @alt_fixtures
+
+  %x{bundle exec rake spec_prep}
+}
+
+require 'spec_helper_acceptance'
+
+hosts.each do |host|
+  describe 'FIPS pre-installed' do
+    after(:suite) do
```

Comment:
@trevor-vaughan This spewed an warning:
WARNING: `after(:suite)` hooks are only supported on the RSpec configuration object. This `after(:suite)` hook, registered on an example group, will be ignored. Called from /var/opt/enemsick/orep/rubygem-simp-beaker-helpers/spec/acceptance/suites/fips_from_fixtures/00_default_spec.rb:44:in `block (2 levels) in <top (required)>'.
WARNING: `after(:suite)` hooks are only supported on the RSpec configuration object. This `after(:suite)` hook, registered on an example group, will be ignored. Called from /var/opt/enemsick/orep/rubygem-simp-beaker-helpers/spec/acceptance/suites/fips_from_fixtures/00_default_spec.rb:44:in `block (2 levels) in <top (required)>'.